egs/swbd/s5c/local/chain/multi_condition/run_tdnn_7b.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=1
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
```

Comment:
OK, I see that by default this uses the speed-perturbed data as input.  you might want to have a comment at the top that clarifies this.